execution_engine/src/core/engine_state/genesis.rs

```diff
@@ -799,7 +797,19 @@ where
 
         let (_, mint_hash) = self.store_contract(access_key, named_keys, entry_points);
 
-        self.protocol_data = ProtocolData::partial_with_mint(mint_hash);
+        {
+            let mut partial_registry = BTreeMap::<String, ContractHash>::new();
```

Comment:
```suggestion
            let mut partial_registry = BTreeMap::new();
```

(or see my other comment about a type alias)

Reply:
Applies suggestion